examples/dti_prediction/README.md

```diff
@@ -0,0 +1,40 @@
+# Drug-Target Interaction: DeepDTA
+
+### 1. Description
+Drug-target interaction is a substantial research area in the field of drug discovery. It refers to predict the binding
```

Comment:
refers to predict: the "to" here is prep (like in happy b-day to you) so you should not put a verb directly after it, more proper to say "refers to predicting". 

Try to use some pycharm plug-in to help you improve your English, e.g. https://plugins.jetbrains.com/plugin/12175-grazie https://plugins.jetbrains.com/plugin/6606-grammar-kit

Use Grammarly for other purposes.

Reply:
You can also google "it refers to " to see how it is used commonly.